packages/babel-plugin-proposal-class-properties/test/fixtures/private-loose/native-classes/output.js

```diff
@@ -18,13 +18,12 @@ class Foo {
 
 var _foo = babelHelpers.classPrivateFieldLooseKey("foo");
 
+var _bar = babelHelpers.classPrivateFieldLooseKey("bar");
+
 Object.defineProperty(Foo, _foo, {
   writable: true,
   value: "foo"
 });
-
-var _bar = babelHelpers.classPrivateFieldLooseKey("bar");
```

Comment:
This is the only change in the output, because I merged the logic for static and instance properties, but it shouldn't be observable.